README.md

```diff
@@ -25,639 +25,7 @@ Before you can integrate with the Ravelin mobile SDK for iOS, you will need:
 
 If you have any questions on getting started, please ask us in your Ravelin support channel on Slack.
 
-## Contents
+## Further information
```

Comment:
Can we change the tittle in the first line of the ReadMe file to be "iOS SDK"? ( I can't comment on that line)
This way we don't have two "Getting Started titles.